Trailing comma rule affects rest operator

```typescript
export type IHandler = {
  (
    targetType: ITargetType | null,
    value: IPropValue,
    ...args: IDecoratorArgs,
  ): IPropValue
}
```

```
src/main/ts/index.ts:97:26 - error TS1013: A rest parameter or binding pattern may not have a trailing comma.

97   ...args: IDecoratorArgs,
```

--- a/src/main/js/tslint-config-qiwi.js
+++ b/src/main/js/tslint-config-qiwi.js
@@ -52,6 +52,7 @@ const rules = {
         'typeLiterals': 'never',
       },
       'singleline': 'never',
+      'esSpecCompliant': true,
     },
   ],
 }

--- a/src/test/ts/rules.out
+++ b/src/test/ts/rules.out
@@ -1,1 +1,2 @@
 ERROR: /rules/trailing-comma-fail.ts:3:7 - Missing trailing comma
+ERROR: /rules/trailing-comma-fail.ts:9:17 - Forbidden trailing comma

--- a/src/test/ts/rules/trailing-comma-fail.ts
+++ b/src/test/ts/rules/trailing-comma-fail.ts
@@ -2,3 +2,9 @@ export const foo = {
   a: 1,
   b: 2
 }
+
+export const fnWithRest = (
+  a: string,
+  b: number,
+  ...rest: any[],
+) => console.log(a, b, ...rest)

--- a/src/test/ts/rules/trailing-comma.ts
+++ b/src/test/ts/rules/trailing-comma.ts
@@ -2,3 +2,9 @@ export const foo = {
   a: 1,
   b: 2,
 }
+
+export const fnWithRest = (
+  a: string,
+  b: number,
+  ...rest: any[]
+) => console.log(a, b, ...rest)

